### PR TITLE
use Grid Layout in Dribbble extension

### PR DESCRIPTION
**Pull request by gebeto**
## Description

 - add `Grid` layout
 - add Details view
 - add more metadata (likes, views)
 - add ability to open author in browser

## Screencast

https://user-images.githubusercontent.com/13422799/172716890-ea2feffc-8b86-4ca1-9e27-77e0a2bd6931.mov

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @gebeto 👋 

Thanks for your update! It looks really good, I only have some minor corrections and suggestions for you 😄 

Could we remove | xxx from the Dropdown items, it should be enough with the section above and will look crisper:

CleanShot 2022-06-10 at 08 59 38@2x

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/dribbble/src/index.tsx [hunk 1]
```diff
@@ -17,73 +23,92 @@ export default function Command() {
...
-![Illustration](${item.image})
-          `}
-            />
-          }
+          content={{ tooltip: item.title, value: { source: item.image } }}
+          title={`${item.title}`}
           actions={
             <ActionPanel>
+              <Action.Push
+                title="More info"
```
[the remark this answers is not in the document]
Comment (gebeto) [reply extensions/dribbble/src/index.tsx]: Title was fixed and added Icon 🙌
Screenshot 2022-06-10 at 12 53 13

**Comment by pernielsentikaer** [comment]
Hi @gebeto 👋 

You might want to leave out the `tintColor` since it looks bad with white apperance
CleanShot 2022-06-10 at 11 56 33@2x

**Comment by gebeto** [comment]
@pernielsentikaer  Oh yeah, thanks!

But I have an issue, if I do not pass tintColor, it is black

So I've used dynamic tintColor:

https://github.com/raycast/extensions/pull/1937/files#diff-da66ebcea774d83320552d706b906b5b28a9cc73289fb06f5dafbd2a2d8e2572R83-R86

**Comment by gebeto** [comment]
@pernielsentikaer Are there some requirements to SVGs for correctly applying default tint color?

**Comment by pernielsentikaer** [comment]
> @pernielsentikaer Are there some requirements to SVGs for correctly applying default tint color?

Seems to be correct now in both light and dark mode, do you face any issues?

**Comment by gebeto** [comment]
> > @pernielsentikaer Are there some requirements to SVGs for correctly applying default tint color?
> 
> Seems to be correct now in both light and dark mode, do you face any issues?

No any issues, now it works correct

**Review by pernielsentikaer — APPROVED** [review]
Awesome @gebeto 🔥

Let's get this beauty out! 💪

**Pull request merged**